core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -146,7 +148,7 @@ object ConsoleProducer {
       .describedAs("size")
       .ofType(classOf[java.lang.Integer])
       .defaultsTo(200)
-    val messageSendMaxRetriesOpt = parser.accepts("message-send-max-retries", "Brokers can fail receiving the message for multiple reasons, and being unavailable transiently is just one of them. This property specifies the number of retires before the producer give up and drop this message.")
+    val messageSendMaxRetriesOpt = parser.accepts("message-send-max-retries", "(Deprecated) Brokers can fail receiving the message for multiple reasons, and being unavailable transiently is just one of them. This property specifies the number of retires before the producer give up and drop this message.")
```

Comment:
@hachikuji @cmccabe @vvcephei -- I think we should deprecate this flag (guess we should update the KIP accordingly) -- Thoughts?

Reply:
Sounds good to me.

Reply:
Updated the KIP and sent a mail to the VOTE thread on the dev list.